src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Microsoft.Extensions.AI.Evaluation.Console.csproj

```diff
@@ -24,13 +24,16 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="Azure.Storage.Files.DataLake" />
```

Comment:
Does the data lake package reference not flow from the Reporting.Azure project reference below?

Reply:
We directly construct a DataLakeFileSystemClient, so we need the reference.

Reply:
Interesting, my assumption was that we won't need to add it explicitly here since we didn't say `PrivateAssets` in the `PackageReference` in the referenced MEAI.E.Reporting.Azure project...